test_conformance/images/kernel_image_methods/test_2D.cpp

```diff
@@ -100,6 +108,10 @@ int test_get_image_info_single( cl_context context, cl_command_queue queue, imag
 
     const char* channelTypeName = GetChannelTypeName( imageInfo->format->image_channel_data_type );
     const char* channelOrderName = GetChannelOrderName( imageInfo->format->image_channel_order );
+    const char *image_access_qualifier =
+        (flags == CL_MEM_READ_ONLY) ? "read_only" : "write_only";
+    const char *cl_khr_3d_image_writes_enabler =
+        "#pragma OPENCL EXTENSION cl_khr_3d_image_writes : enable";
```

Comment:
Adding this unconditionally may cause a problems for devices that don't support 3D image writes (e.g. 1.2 and 3.0 devices that don't support this extension). This should only be added when testing 3D images with `CL_MEM_WRITE_ONLY`, and that test should be skipped (somewhere earlier) if the device doesn't support this functionality.